lib/jit_preloader/active_record/base.rb

```diff
@@ -33,7 +33,7 @@ def preload_scoped_relation(name:, base_association:, preload_scope: nil)
       records.each do |record|
         record.jit_preload_scoped_relations ||= {}
         association = record.association(base_association)
-        record.jit_preload_scoped_relations[name] = preloader_association.records_by_owner[record]
+        record.jit_preload_scoped_relations[name] = preloader_association.records_by_owner[record] || []
```

Comment:
If the ` preloader_association.records_by_owner[record]` is an `ActveRecord` relation when present, should we return [`none`](https://api.rubyonrails.org/v6.0.0/classes/ActiveRecord/QueryMethods.html#method-i-none) when it's nil?

Reply:
Good idea! Just double-checked and it's an array, when present, though unfortunately.